WIP-CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ### Added
 
+- Add support for self-managed CRDs (Beta)
+  - Add support for sealedsecrets and mongodb
+- Add end-user service account kube-config for devs
```

Comment:
Who is the End User here? I'm a bit unsure we use this term according to our [glossary](https://elastisys.io/compliantkubernetes/glossary/#end-user).